hazelcast/src/main/java/com/hazelcast/cache/impl/CacheSplitBrainHandlerService.java

```diff
@@ -124,6 +127,18 @@ public Runnable prepareMergeRunnable() {
         return new CacheMerger(recordMap);
     }
 
+    // overridden on ee
+    protected List<Iterator<ICacheRecordStore>> iteratorsOf(CachePartitionSegment segment) {
+        List<Iterator<ICacheRecordStore>> iterators = new LinkedList<Iterator<ICacheRecordStore>>();
+        iterators.add(segment.recordStoreIterator());
```

Comment:
minor: `Collections.singletonList(segment.recordStoreIterator())` might be used.